webapp/src/pages/BsmMap.js

```diff
@@ -129,6 +121,15 @@ function BsmMap(props) {
     { value: 60, label: "60 minutes" },
   ];
 
+  useEffect(() => {
+    if (!startBsmDate) {
+      dateChanged(new Date(), "start");
+    }
+    if (!endBsmDate) {
+      dateChanged(new Date(), "end");
+    }
+  }, []);
```

Comment:
Missing dependency array entries

Reply:
There is no warning shown for this one, so I think it is fine?

Reply:
> I'm not sure if it is just on this PR or an on-going issue but when i remove multiple RSUs from my organization, the table attempts to delete a single RSU, then refresh and repeat. This gives the table a bit of a stutter when deleting multiple records and is a bit confusing. We should instead allow removing multiple items while displaying the loading indicator then refresh a single time.

I just updated the rsuDeleteMultiple function to delete them all at once, instead of one at a time. I made the same change to any similar methods, specifically rsuAddMultiple, userDeleteMultiple, and userAddMultiple.

Reply:
@payneBrandon I for some reason replied to your comment above in this thread, oops.